src/OpenTelemetry/CHANGELOG.md

```diff
@@ -26,6 +26,10 @@
   them to supported data types (long for int/short, double for float). For
   invalid attributes we now throw an exception instead of logging an error.
   ([#1720](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1720))
+* Merging "this" resource with another previously kept "this" resource's
```

Comment:
This sentence is crashing my brain 🤣 Did you mean to use "this" twice?

Reply:
Haha I did, but you're right that it is confusing and on a re-read it took me many tries to comprehend, I'll reword it.